docs/slides/innersource-summit.md

```diff
@@ -0,0 +1,231 @@
+
+#### InnerSource Metrics include: 
+
+NEEDS EXAMPLE HERE 
+
+<!-- KEY MESSAGE 12:  
+Are the the metrics being collected, why, and give examples of different companies?
+-->
+
+---
+<!--- Slide 13 --->
+
+Why did I from contributing to collecting examples of how different companies are doing these metrics?
+
+<!-- KEY MESSAGE 12:  
+ (Jeff)
+ 
+ NEEDS POPULATED
+-->
+
+
+--- 
+<!--- Slide 14 --->
+
+## How the ISPO working group operates
+
+Deliver valuable products through collaborative efforts, accommodating participants' availability and interest 
+
+<!-- KEY MESSAGE 14: How: How the working group operates, and how the audience can contribute in ways that maximize the benefits of the working group for their own needs -->
+
+--- 
+<!--- Slide 15 --->
+
+## When:
+
+#### Asynchronously
+- Slack channel
+- GitHub repositories 
+- GitHub-based Kanban board
+
+#### Synchronously
+- Biweekly working group meetings 
+
+<!-- KEY MESSAGE 15: People can attend each biweekly meeting, or work asynchronously, or drop in when available. -->
+
+--- 
+<!--- Slide 16 --->
+
+## How we work
+
+1. Kanban board _(GitHub)_
+2. Peer review _(GitHub)_
+3. Focus on external products useful to others
+  - Markdown books, website pages, and other things that can be linked to from inside company resources or ingested into them.
+
+<!-- KEY MESSAGE 16: All projects within the working group undergo peer review as they progress through the Kanban board -->
+
+---
+<!---  Slide 17: Closing Point of View --->
+### The ISPO working group can be your force multiplier!
+
+- Produce content at lower cost collaboratively
+- Get feedback from deeply experienced people in your niche
+
+<!-- KEY MESSAGE 17: "We want people to take the work in their company backlog and align it to what is being contributed in this working group.
+E.g. in peoples' company backlog there are tickets that talk about them working and building this out in the InnerSource Commons." -->
+
+---
+<!---  Slide 18: Specific Action --->
+
+Attend an ISPO working group meeting
+
+Get meeting invites by adding an issue: https://github.com/InnerSourceCommons/ispo-working-group/issues/new
+
+<!-- KEY MESSAGE 18: DO THIS ONE THING RIGHT NOW -->
+
+
+---
+<!---  Slide 19: Benefits of Action --->
+
+The working group has the same benefits as open source. 
+
+
+Build it together ... work together ... reduce duplication.
+
+<!-- "You should know this already - same benefits as open source.
+Build it together ... work together ... reduce duplication.
+Run our InnerSource programs as a collective open source project." -->
+
+---
+<!---  Slide 19: Closing Statement --->
+
+## Collaborate with us!
+
+- GitHub repository https://github.com/InnerSourceCommons/ispo-working-group 
+- Kanban board: https://github.com/orgs/InnerSourceCommons/projects/4 
+- Join our slack group: https://innersourcecommons.org/community/#ispo 
+- Get meeting invites: https://github.com/InnerSourceCommons/ispo-working-group/issues/new
```

Comment:
GitHub must have made some kind of update because now our "join" template doesn't work: https://github.com/InnerSourceCommons/ispo-working-group/blob/main/.github/ISSUE_TEMPLATE/join-wg.yml

Reply:
Here's the error in the template.

I haven't used this feature, but maybe the field could be a text field asking for name?

<img width="873" alt="image" src="https://github.com/InnerSourceCommons/ispo-working-group/assets/776901/5af58854-ace7-4709-aa9d-88b3603622f5">